service/locker/distributed.go

```diff
@@ -0,0 +1,106 @@
+package locker
```

Comment:
Following the already established pattern the file should be called `kubelock_locker.go` and the implementation should be called `KubeLockLocker`.

Reply:
+1 on this.

Reply:
done